src/main/scala/firrtl/annotations/PresetAnnotations.scala

```diff
@@ -0,0 +1,38 @@
+// See LICENSE for license details.
+
+package firrtl
+package annotations
+
+/**
+  * Transform all registers connected to the targeted AsyncReset tree into bitstream preset registers
+  * Impacts all registers connected to any child (cross module) of the target AsyncReset
+  *
+  * @param target ReferenceTarget to an AsyncReset
+  */
+class PresetAnnotation(
+  target: ReferenceTarget
+) extends firrtl.transforms.DontTouchAnnotation(target) {
+  override def duplicate(n: ReferenceTarget) = this.copy(target = n)
+}
```

Comment:
#1433 is really great ! To be honnest I hesitated to integrate a similar change within this PR but I refrained myself since it is a completely different subject. Thanks for the pointer and review, I will address your comments today